src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -3270,6 +3836,9 @@ public Result generateResult(Submission submission, User assessor) {
         return result;
     }
 
+    /**
+     * Moved to {@link ParticipationUtilService#generateComplaintAndResponses(String, int, int, int, boolean, Result, User)}
```

Comment:
I'm not sure if methods related complaints belong in the ParticipationUtilService. Assessment is part of the ExerciseUtilService so it might make sense to also have complaints there. Or what do you think?

Reply:
Discussed: I will move them into a new ComplaintUtilService